docs_src/examples/index.md

```diff
@@ -23,8 +23,7 @@ The tabs below provide a listing (may be partial based on latest updates) for re
 === "Security"
     |Example|Location|
     |---|---|
-    |Docker Swarm, remote device service via overlay network|[Github - Docker Swarm](https://github.com/edgexfoundry/edgex-examples/tree/v2.2.0/security/remote_devices/docker-swarm)|
-    |SSH Tunneling, remote device service via SSH tunneling|[Github - SSH Tunneling](https://github.com/edgexfoundry/edgex-examples/tree/v2.2.0/security/remote_devices/ssh-tunneling)|
```

Comment:
Why did these two links get removed?

Reply:
These files got removed recently. I switched it to the only security example I could find: https://github.com/edgexfoundry/edgex-examples/tree/v2.2.0/security/remote_devices/spiffe_and_ssh
@bnevis-i can polish it up after it passes and we enable the htmlproofer

Reply:
Sounds good.